examples/cassandra/README.md

```diff
@@ -467,24 +470,32 @@ You may need to disable config file validation, like so:
 $ kubectl create -f examples/cassandra/cassandra-daemonset.yaml --validate=false
```

Comment:
No, just kicked off a GKE cluster which is 1.1.  We should update the Prereq section to explicitly state 1.2 is required.